### PR TITLE
XS✔ ◾ update rule how to share file folder in sharepoint

### PR DESCRIPTION
**Pull request by Tanya-Leahy**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

✏️[User Story 53994](https://ssw2.visualstudio.com/SSW.SharePoint/_workitems/edit/53994): 👨‍👩‍👧‍👦 Teams - Update Rule to Share Links

> 2. What was changed?

✏️Updated and done differently to include how ways of sharing a document can change permissions.

> 3. Did you do pair or mob programming (list names)?

✏️No, just updated the rule myself.

### CODE REVIEW

File: rules/how-to-share-a-file-folder-in-sharepoint/rule.md [hunk 1]
```diff
@@ -10,29 +10,52 @@ authors:
...
 
+You may be tempted to share a link via the Share email invite. In doing so, the sharing permissions default to 'can view', which is fine if you are sharing the file externally. If you are sharing to someone who already has access to the file, it breaks permissions on the item you're sharing and creates a unique permission set for whoever you share it with. This means that a person who already had access to a file may find that using the shared link grants them different permissions. This makes it very difficult for SharePoint admins to maintain permissions!
+
+::: bad  
+![Figure: Bad example - Sharing URL via Share email invite](sharepoint-share-email-invite.png)  
+
+:::
+
+::: bad  
+![Figure: Bad example - Mulitple unique permission sets for a document library](sharepoint-file-unique-permissions.png)
```
Comment (leahy268) [inline rules/how-to-share-a-file-folder-in-sharepoint/rule.md:34]: Remove this image as it is already in the other rule and the associated file

**Review by leahy268 — CHANGES_REQUESTED** [review]
Remove the extra file and the entire ::Bad section as per the comments

File: rules/how-to-share-a-file-folder-in-sharepoint/rule.md (hunk 1 above)
Comment (Tanya-Leahy) [reply rules/how-to-share-a-file-folder-in-sharepoint/rule.md]: Done

**Review by leahy268 — APPROVED** [review]
LGTM

**Pull request merged**